spiffworkflow-frontend/src/components/ProcessInstanceListTable.tsx

```diff
@@ -1868,72 +515,21 @@
     );
   };
 
-  const reportSearchComponent = () => {
-    if (showReports) {
-      const columns = [
-        <Column sm={4} md={8} lg={16}>
-          <ProcessInstanceReportSearch
-            onChange={processInstanceReportDidChange}
-            selectedItem={processInstanceReportSelection}
-            selectedReportId={searchParams.get('report_id')}
-            handleSetSelectedReportCallback={setProcessInstanceReportSelection}
-          />
-        </Column>,
-      ];
-      return (
-        <Grid className="with-tiny-bottom-margin" fullWidth>
-          {columns}
-        </Grid>
-      );
-    }
-    return null;
-  };
-
-  const tableTitleLine = () => {
-    if (!showLinkToReport && !headerElement) {
-      return null;
-    }
-    let filterButtonLink = null;
-    if (showLinkToReport && processInstances.length > 0) {
-      filterButtonLink = (
-        <Column
-          sm={{ span: 1, offset: 3 }}
-          md={{ span: 1, offset: 7 }}
-          lg={{ span: 1, offset: 15 }}
-          style={{ textAlign: 'right' }}
-        >
-          <Button
-            data-qa="process-instance-list-link"
-            renderIcon={ArrowRight}
-            iconDescription="View Filterable List"
-            hasIconOnly
-            size="md"
-            onClick={() =>
-              navigate(`/process-instances?report_hash=${reportHash}`)
-            }
-          />
-        </Column>
-      );
-    }
-    if (!headerElement && !filterButtonLink) {
-      return null;
-    }
-    return (
-      <>
-        <Column
-          sm={{ span: 3 }}
-          md={{ span: 7 }}
-          lg={{ span: 15 }}
-          style={{ height: '48px' }}
-        >
-          {headerElement}
-        </Column>
-        {filterButtonLink}
-      </>
+  const errors: string[] = [];
+  if (additionalReportFilters && reportMetadata) {
+    errors.push(
+      'Both reportMetadata and additionalReportFilters were provided. ' +
+        'It is recommended to only use additionalReportFilters with reportIdentifier and to specify ALL filters in reportMetadata if not using reportIdentifier.'
     );
-  };
+  }
+  if (reportIdentifier && reportMetadata) {
+    errors.push(
+      'Both reportIdentifier and reportMetadata were provided. ' +
+        'You must use one or the other.'
+    );
```

Comment:
We display these errors through the normal the normal error displaying since this component can be configured in other ways that need to be more apparent in some cases outside of a normal development environment.